templates/page.html

```diff
@@ -345,41 +346,23 @@ <h3>Current</h3>
         <div class="card">
           <div class="card-body">
             <h5 class="card-title">
-              <a href="https://github.com/orgs/reanahub/projects/38" class="card-link">
-                Snakemake integration
+              <a href="https://github.com/orgs/reanahub/projects/40" class="card-link">
+                ATLAS-pMSSM-Basics
```

Comment:
Not sure whether we should show this here. Also, I need input on items to put in Current, Near-term and Future.

Reply:
Yeah, it might be a bit ATLAS specific, while here it would be good to point only to all the goodies that comes out of the project.  A quick fix would be to change the title to "Workflow scalability" and not to link it to the GitHub board.  After all these are just high-level boxes... We can also revive the roadmap boxes right after the release in calmer times.